reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/server/JavaServerDriver.java

```diff
@@ -125,22 +131,23 @@ public RequestHandler parse() {
             Tuple<String, String> initialPayload = new Tuple<>(ByteBufferUtil.toUtf8String(payload.getData()),
                     ByteBufferUtil.toUtf8String(payload.getMetadata()));
             String marble = requestSubscriptionMarbles.get(initialPayload);
-            System.out.println("Subscription " + initialPayload.getK() + " " + initialPayload.getV());
+            System.out.println(ANSI_CYAN + "Received Subscription " + initialPayload.getK()
+                    + " " + initialPayload.getV() + ANSI_RESET);
             if (marble != null) {
                 ParseMarble pm = new ParseMarble(marble, s);
                 s.onSubscribe(new TestSubscription(pm));
                 new ParseThread(pm).start();
             }
         }).withRequestChannel(payloadPublisher -> s -> { // design flaw
             try {
-                System.out.println("Channel");
                 TestSubscriber<Payload> sub = new TestSubscriber<>();
                 payloadPublisher.subscribe(sub);
                 // want to get equivalent of "initial payload"
                 //sub.request(1); // first request of server is implicit, so don't need to call request(1) here
                 sub.awaitAtLeast(1);
                 Tuple<String, String> initpayload = new Tuple<>(sub.getElement(0).getK(), sub.getElement(0).getV());
-                System.out.println(initpayload.getK() + " " + initpayload.getV());
+                System.out.println(ANSI_CYAN + "Received Channel" + initpayload.getK()
```

Comment:
That makes sense, so it'll basically be some sort of logger but for this specifically